server/api/image_uploader.py

```diff
@@ -1,4 +1,6 @@
 import cloudinary.uploader
+from config import CLOUDINARY_NAME
 
-def image_uploader(image_url, image_preset, cloud_name):
-    return cloudinary.uploader.unsigned_upload(image_url, image_preset, cloud_name=cloud_name)['secure_url']
+def replace_cloudinary_image(image_url, preset):
```

Comment:
We could probably just use PRODUCT_IMG_PRESET as a default parameter here and not have to import/use it all the places this is called